ui/core.js

```diff
@@ -157,13 +157,13 @@ $.extend( $.expr[ ":" ], {
 		},
 
 	focusable: function( element ) {
-		return focusable( element, !isNaN( $.attr( element, "tabindex" ) ) );
+		return focusable( element, $.attr( element, "tabindex" ) != null );
```

Comment:
We used to actually test for invalid values: https://github.com/jquery/jquery-ui/commit/f80d9eb465e428b3900bc0b741392f14ecd859f0#diff-47738b779c41035dcc321cc78634c6a5R87, but we dropped those tests four years ago.